### PR TITLE
Mobile site structure

### PR DESCRIPTION
**Pull request by SamBellDev**
Content not complete but layout for mobile complete.

### CODE REVIEW

**Comment by Albertoroldanq** [comment]
Good job Sam! 🙌  To be honest at first I thought I didn't have anything to comment so I had to set my mindset much more strict in order to find "bugs", so don't panic even if you see a lot of text, most of them are only minor issues. And please don't misunderstand my tone 😆

HTML:
- H1-> the h1 element is grouping several elements inside (img and h2) need to move the closing just below the text of the H1
- To be more picky, you could use <section> to separate about me and projects? 
- Inside projects-container div can delete one level of indentation.
- I think that <main> should be closed before opening <footer>?
- If you want to link the social media icons you can wrap them with an <a>
- H2 "Contact me": closing tag in the same line whilst you have the rest of closing tags for the headings in a separate line.
- Indentation in the div socials container.

CSS:
- if you only use .heading for headings, maybe you don't need to set display:block (since I think it is the value by default)
- Portfolio and About me are both H2 but the sizes are different. I don't see any css code that sets their font-size, they probably look different because of the bug in the HTML I mentioned about the H1

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+					 Future project
+				 </h3>
+				 <img src="idea.jpg" alt="Thumbnail of project 2">
+				 <p>
+					 This project is still a whimisical thought in my brain.
+				 </p>
+		 </div>
+	 </div>
+	 <footer id="Contact me">
+		 <h2 id="Contact me" class="heading">Contact me</h2>
```
Comment (Albertoroldanq) [inline portfolio.html:62]: Maybe you can put the closing tag in a separate line, like the rest of your headings.

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	 <div class="title-container">
+			 <h1 id="home" class="heading">
+				 Sam Bell
+			 </h1>
+			 <img src="me.jpg" alt="Portrait of Sam Bell">
+			 <h2 class="heading">
+				 Portfolio
+			 </h2>
+	 </div>
+	 <div>
```
Comment (Albertoroldanq) [inline portfolio.html:25]: you could use <section> to be more semantic I guess, same with the projects

File: portfolio.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+		 </p>
+	 </div>
+	 <div class="container">
+		 <div>
+			 <h2 id="Projects" class="heading">
+				 Projects
+			 </h2>
+		 </div>
+		 <div class="projects-container">
+				 <h3 class="heading">
```
Comment (Albertoroldanq) [inline portfolio.html:43]: too much indentation I think

File: portfolio.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+		 </h2>
+		 <p>
+			 I want to become a Full Stack Web Developer and I'm training with iO academy to get there. I want my code to look so clean you need sunglasses to read it, but that might be a stretch goal.....
+		 </p>
+		 <p>
+			 I'm a climber, gardener, mountain biker, and scuba diver in my free time. I recently hiked the three peaks and am mid-way through landscaping my garden.
+		 </p>
+	 </div>
+	 <div class="container">
+		 <div>
```
Comment (Albertoroldanq) [inline portfolio.html:37]: I am not sure if it is necessary to group the h2 in a div

File: portfolio.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+			 </a>
+			 <a href="https://twitter.com/SamBelldev" target="_blank" rel="noopener noreferrer">
+				 <img src="twitter.png" alt="Twitter logo">
+			 </a>
+			 <a href="mailto: sambelldev@tuta.io" target="_blank" rel="noopener noreferrer">
+				 <img src="email.png" alt="email logo">	
+			 </a>
+		 </div>	  	 
+	 </footer>
+ </main>
```
Comment (Albertoroldanq) [inline portfolio.html:79]: Not sure if main should be closing before opening the footer

File: portfolio.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head> 
+  <title>Sam Bell portfolio page</title> 
+  <link rel="stylesheet" type="text/css" href="normalize.css">
+  <link rel="stylesheet" type="text/css" href="portfolio.css">
+ </head> 
+ <body>
```
Comment (Albertoroldanq) [inline portfolio.html:8]: the indentation inside head is different than the body, probably you don't need as much space in the body

File: portfolio.html [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+			 <a href="https://twitter.com/SamBelldev" target="_blank" rel="noopener noreferrer">
+				 <img src="twitter.png" alt="Twitter logo">
+			 </a>
+			 <a href="mailto: sambelldev@tuta.io" target="_blank" rel="noopener noreferrer">
+				 <img src="email.png" alt="email logo">	
+			 </a>
+		 </div>	  	 
+	 </footer>
+ </main>	 
+ </body>
```
Comment (Albertoroldanq) [inline portfolio.html:80]: I think the html closing tag is missing after the body tag

File: portfolio.css [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+.projects-container {
+	display: flex;
+	flex-flow: column nowrap;
+	align-items: center;
+}
+
+.socials-container {
+	display: flex;
+	flex-flow: row nowrap;
```
Comment (Albertoroldanq) [inline portfolio.css:59]: row is the default value, so maybe you can simplify by just setting the flex-wrap property

File: portfolio.css [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+  background-color: #ffffff;
+}
+
+.nav-container{
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-around;
+}
+
+.title-container {
```
Comment (Albertoroldanq) [inline portfolio.css:45]: .you can add the .projects-container selector in this same line with a "," since they both have the settings

**Review by Albertoroldanq — COMMENTED** [review]
Ok, I think that's everything, some comments I have added e.g. about grouping some css selectors are just because it's supposed to be best practice, but for me looks good, is tidy and easy to understand. And the rest of comments as you can see are just minor issues, everything works fine!

File: portfolio.html (hunk 1 above)
Comment (SamBellDev) [reply portfolio.html:62]: Done

File: portfolio.html (hunk 2 above)
Comment (SamBellDev) [reply portfolio.html:25]: Changed

File: portfolio.html (hunk 5 above)
Comment (SamBellDev) [reply portfolio.html:79]: changed to close before footer.

File: portfolio.html (hunk 7 above)
Comment (SamBellDev) [reply portfolio.html:80]: added

File: portfolio.html (hunk 4 above)
Comment (SamBellDev) [reply portfolio.html:37]: Thanks for tidying up! Must have been a hangover from previous design iteration.

File: portfolio.css (hunk 8 above)
Comment (SamBellDev) [reply portfolio.css:59]: I like being explicit for readability.

File: portfolio.css [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+	align-items: center;
+}
+
+.socials-container {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: center;
+}
+
+.heading {
```
[the remark this answers is not in the document]
Comment (SamBellDev) [reply portfolio.css:63]: I didn't think h elements were display block by default. I could be wrong. will look. You're right. I was thinking of p

[the remark this answers is not in the document]
Comment (SamBellDev) [reply portfolio.css:63]: changed

File: portfolio.css (hunk 9 above)
Comment (SamBellDev) [reply portfolio.css:45]: Thanks for the tip. Changed

File: portfolio.html (hunk 6 above)
Comment (SamBellDev) [reply portfolio.html:8]: I like the space for readability

File: portfolio.html (hunk 3 above)
Comment (SamBellDev) [reply portfolio.html:43]: changed

**Review by Albertoroldanq — APPROVED** [review]
Well done

**Pull request merged**